packages/old-import/src/widgets/definitions/rss.ts

```diff
@@ -11,5 +11,6 @@ export type OldmarrRssDefinition = CommonOldmarrWidgetDefinition<
     sortByPublishDateAscending: boolean;
     sortPostsWithoutPublishDateToTheTop: boolean;
     maximumAmountOfPosts: number;
+    hideDescription: boolean;
```

Comment:
Here we don't want to add it as the old version did not contain this option:
https://github.com/ajnart/homarr/blob/dev/src/widgets/rss/RssWidgetTile.tsx